config/canonical.json

```diff
@@ -31382,8 +31382,8 @@
     "count": 152,
     "tags": {
       "brand": "Tesco Metro",
-      "brand:wikidata": "Q487494",
-      "brand:wikipedia": "en:Tesco",
+      "brand:wikidata": "Q57551648",
+      "countryCodes": ["gb"],
```

Comment:
This value does not belong into the tags object. You have to add it below the value of count:*